src/Sylius/Bundle/ApiBundle/Validator/Constraints/ChosenPaymentMethodEligibility.php

```diff
@@ -18,7 +18,10 @@
 final class ChosenPaymentMethodEligibility extends Constraint
 {
     /** @var string */
-    public $message = 'sylius.payment_method.not_exist';
+    public $paymentMethodNotAvailable = 'sylius.payment_method.not_available';
```

Comment:
```suggestion
    public $notAvailable = 'sylius.payment_method.not_available';
```

Should be pretty clear it's connected to the payment method.